src/lib.rs

```diff
@@ -28,6 +33,32 @@ pub use crate::{
     mint::{Mint, MintRequest, MintTransaction},
 };
 
+impl From<[u8; 32]> for Hash {
+    fn from(val: [u8; 32]) -> Hash {
+        Hash(val)
+    }
+}
+
+impl Distribution<Hash> for Standard {
+    fn sample<R: Rng + ?Sized>(&self, rng: &mut R) -> Hash {
+        Hash(rng.gen())
+    }
+}
```

Comment:
This seems a bit weird, is it required? (apart from that, great addition to view Dbcs like this)

Reply:
It's only used when fuzzing DBC's in testing.

@dan-da lets add a `#[cfg(test)]` in front of this impl as to not pollute the public API

Reply:
k, will do

Reply:
done.